opengrok-indexer/build.xml

```diff
@@ -121,7 +121,7 @@ Copyright (c) 2007, 2016, Oracle and/or its affiliates. All rights reserved.
               defaultexcludes="no"/>
         <zip destfile="${build.test.classes.dir}/org/opensolaris/opengrok/history/repositories.zip" 
                 basedir="${test.repositories}"
-                excludes="mercurial/hg/**, mercurial/hgignore, git/git"
+                excludes="mercurial/hg/**, mercurial/hgignore, git/git/**"
```

Comment:
For me the compiled test-classes still contained the both git and .git directory which caused a null revision error for getting annotations from the git repository (as they aren't tracked by git but still are in the repository).

This exact same line is in the root build.xml with these `**` at the end.